vscode/src/client/analyzerClient.ts

```diff
@@ -106,21 +130,19 @@ export class AnalyzerClient {
     // Define the initialize request parameters if needed
     const initializeParams = {
       process_id: null,
-      kai_backend_url: "0.0.0.0:8080",
+      kai_backend_url: getConfigKaiBackendURL(),
       root_path: vscode.workspace.workspaceFolders![0].uri.fsPath,
-      log_level: "debug",
+      log_level: getConfigLogLevel(),
       log_dir_path: this.kaiDir,
       model_provider: {
-        provider: "ChatOpenAI",
+        provider: getConfigKaiProviderName(),
         args: {
-          model: "gpt-3.5-turbo",
-          // parameters: {
-          //   max_new_tokens: 2048,
-          // },
+          model: getConfigKaiProviderModel(),
+          parameters: getConfigKaiProviderParameters(),
```

Comment:
Something seems to fail when passing in the parameters argument for me:

```ERROR - 2024-12-03 13:50:01,363 - kai.kai.reactive_codeplanner.task_manager.task_manager - Thread-1 - [task_manager.py:112 - supply_result()] - Encountered errors: ["Completions.create() got an unexpected keyword argument 'parameters'"]```

Reply:
Ian which kai rpc binary are you using?  This is something I can't test w/o linux binaries.

Reply:
Yeah that is the one @sjd78. Using an m1 mac luckily.